WSRequestHandler.cpp

```diff
@@ -56,6 +56,8 @@ WSRequestHandler::WSRequestHandler(QWebSocket* client) :
     messageMap["SetSceneItemPosition"] = WSRequestHandler::HandleSetSceneItemPosition;
     messageMap["SetSceneItemTransform"] = WSRequestHandler::HandleSetSceneItemTransform;
     messageMap["SetSceneItemCrop"] = WSRequestHandler::HandleSetSceneItemCrop;
+    // This method name needs work. Perhaps just "GetSceneItemProperties"
```

Comment:
Yeah, saying `Scene` twice seems redundant.

Reply:
Docs should probably say that this returns properties that are specific to a given scene and item (mostly not generic to an item)